ServiceRequest/UKCore-ServiceRequest-MichaelJonesDYPDRequest-Example.json

```diff
@@ -38,33 +38,42 @@
 	"status": "active",
 	"intent": "order",
 	"category": [
-		{ 
+		{
```

Comment:
Can you remove this completely? The malignant neoplasm code is not a category, and we already have a code for the servicerequest.